### PR TITLE
Remove SND options from Staff/Part Properties and hairpin playback popup

### LINKED ISSUE #11417: Remove option for 'Single Note Dynamics' in the Staff/Part Properties dialog
**Describe the bug**
This option (at the bottom left corner of the dialog) no longer functions, since we have replaced the entire engine and how it works. It was always a pointless option to expose to the user anyway. Good riddance!

image

### PR DESCRIPTION
**Pull request by quinnouyang**
Resolves #11417 

Title. Removed relevant XML from `editstaff.ui` and legacy code from `editstaff.cpp`. Also removed from `hairpinplaybackmodel`.

Noticed quite a bit of non-UI/functionality code surrounding SND, but I don't know if those are still necessary so I let them be.

## Staff/Part Properties
| Before | After |
| ------------- | ------------- |
| image  | image  |

## Hairpin Playback Popup
| Before | After |
| ------------- | ------------- |
| image  | image  |

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: src/inspector/view/qml/MuseScore/Inspector/general/playback/internal/HairpinsExpandableBlank.qml [hunk 1]
```diff
@@ -79,17 +79,6 @@ ExpandableBlank {
             minValue: 0
         }
 
-        CheckBoxPropertyView {
-            id: singleNoteCheckBox
```
Comment (RomanPudashkin) [inline src/inspector/view/qml/MuseScore/Inspector/general/playback/internal/HairpinsExpandableBlank.qml:83]: This id is still used in velocityChangeTypeSection

`navigationRowStart: singleNoteCheckBox.navigation.row + 1`

Comment (quinnouyang) [reply src/inspector/view/qml/MuseScore/Inspector/general/playback/internal/HairpinsExpandableBlank.qml:83]: Got it: replaced the id with the one directly above it. I assume this fixes it, but is there a difference between using `.navigation.row` and `navigationRowEnd`?

Comment (RomanPudashkin) [reply src/inspector/view/qml/MuseScore/Inspector/general/playback/internal/HairpinsExpandableBlank.qml:83]: You should use navigationRowEnd here:

navigationRowStart:  velocityChangeSection.navigationRowEnd + 1

Comment (RomanPudashkin) [reply src/inspector/view/qml/MuseScore/Inspector/general/playback/internal/HairpinsExpandableBlank.qml:83]: >is there a difference between using .navigation.row and navigationRowEnd?

Some controls in Inspector may occupy a range of navigation indexes. navigationRowEnd is needed to indicate the end of these occupied indexes

Comment (quinnouyang) [reply src/inspector/view/qml/MuseScore/Inspector/general/playback/internal/HairpinsExpandableBlank.qml:83]: Thanks, makes sense! Just ammended the commit with `navigationRowEnd`.

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**